packages/core-types/tests/integration/components/field-editors/date-editor-test.js

```diff
@@ -1,34 +1,74 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { render, focus, blur } from '@ember/test-helpers';
+import EmberObject from '@ember/object';
 import hbs from 'htmlbars-inline-precompile';
 
+let input;
+
 module('Integration | Component | field editors/date editor', function(hooks) {
   setupRenderingTest(hooks);
 
+  hooks.beforeEach(() => {
+    input = '.field-editor > input';
+  });
```

Comment:
why do you set that in `beforeEach`? would be easier to make it `const INPUT = '.field-editor > input';`, right?